fix(curriculum): Recipe Tracker improvements (#57694)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5636.md

You should access the `averageRating` property of `recipe2`.

```js
assert.isNotNull(recipe2.averageRating);
```

> You should set the `averageRating` of `recipe2` you accessed to the result of `getAverageRating` with `recipe1.ratings` as its argument.

Reviewer: ```suggestion
You should assign the result of calling `getAverageRating` with `recipe2.ratings` to the `averageRating` property of `recipe2`.
```

> You should set the `averageRating` of `recipe2` you accessed to the result of `getAverageRating` with `recipe1.ratings` as its parameter.

Reviewer 2: ```suggestion
You should set the `averageRating` of `recipe2` you accessed to the result of `getAverageRating` with `recipe2.ratings` as its argument.
```

```js
assert.strictEqual(recipe2.averageRating, getAverageRating(recipe2.ratings));
```

You should access the `totalIngredients` property of `recipe2`.

```js
assert.isNotNull(recipe2.totalIngredients);
```

> You should set the `totalIngredients` of `recipe2` you accessed to the result of `getTotalIngredients` with `recipe2.ingredients` as its argument.

Reviewer: ```suggestion
You should assign the result of calling `getTotalIngredients` with `recipe2.ingredients` to the `totalIngredients` property of `recipe2`.
```

> You should set the `totalIngredients` of `recipe2` you accessed to the result of `getTotalIngredients` with `recipe2.ingredients` as its parameter.

Reviewer 2: ```suggestion
You should set the `totalIngredients` of `recipe2` you accessed to the result of `getTotalIngredients` with `recipe2.ingredients` as its argument.
```

```js
assert.strictEqual(recipe2.totalIngredients, getTotalIngredients(recipe2.ingredients));
```

You should access the `difficultyLevel` property of `recipe2`.

```js
assert.isNotEmpty(recipe2.difficultyLevel);
```

> You should set the `difficultyLevel` of `recipe2` you accessed to the result of `getDifficultyLevel` with `recipe2.cookingTime` as its argument.

Reviewer: ```suggestion
You should assign the result of calling `getDifficultyLevel` with `recipe2.cookingTime` to the `difficultyLevel` property of `recipe2`.
```

> You should set the `difficultyLevel` of `recipe2` you accessed to the result of `getDifficultyLevel` with `recipe2.cookingTime` as its parameter.

Reviewer 2: ```suggestion
You should set the `difficultyLevel` of `recipe2` you accessed to the result of `getDifficultyLevel` with `recipe2.cookingTime` as its argument.
```
